article/app/views/structuredData/BlogPosting.scala

```diff
@@ -1,4 +1,4 @@
-package views.support.structuredData
+package views.structuredData
```

Comment:
Putting this in `views/support` in common was an mistake I think.
`views` subfolders should contain `views` (ie: `.scala.html`)